explainer.md

```diff
@@ -147,13 +152,13 @@ for await (const stream of transport.receiveBidirectionalStreams()) {
 
 ```javascript
 const mediaSource = new MediaSource();
-await new Promise(resolve => mediaSource.addEventListener('sourceopen', () => resolve(), {once: true}));
+await new Promise(resolve => mediaSource.addEventListener('sourceopen', resolve, {once: true}));
 const sourceBuffer = mediaSource.addSourceBuffer('video/webm; codecs="opus, vp09.00.10.08"');
-const transport = new Http3Transport("/video");
-await fetch('http://example.com/babyshark');
-for await (const datagram of transport.receiveDatagrams()) {
+const transport = new WebTransport('/video');
```

Comment:
Interesting. I suppose we can make relative URLs work without too much difficulty. I wonder how useful it is, though.

Reply:
Yeah, we can use https://url.spec.whatwg.org/#concept-url-parser.